yolov6/core/engine.py

```diff
@@ -294,23 +295,24 @@ def after_epoch(self):
                             "2": lrs_of_this_epoch[2],
                         },
                         "val": {
-                            "mAP@0.5": self.evaluate_results[0],
-                            "mAP@0.50:0.95": self.evaluate_results[1],
+                            "mAP_0.50-0.95": self.evaluate_results[0],
+                            "mAP_0.50": self.evaluate_results[1],
+                            "mAP_0.75": self.evaluate_results[2],
+                            "mAP_0.50-0.95_small": self.evaluate_results[3],
+                            "mAP_0.50-0.95_medium": self.evaluate_results[4],
+                            "mAP_0.50-0.95_large": self.evaluate_results[5],
+                            "mAR_0.50-0.95_maxDets-1": self.evaluate_results[6],
+                            "mAR_0.50-0.95_maxDets-10": self.evaluate_results[7],
+                            "mAR_0.50-0.95_maxDets-100": self.evaluate_results[8],
+                            "mAR_0.50-0.95_small": self.evaluate_results[9],
+                            "mAR_0.50-0.95_medium": self.evaluate_results[10],
+                            "mAR_0.50-0.95_large": self.evaluate_results[11]  
```

Comment:
It would be good if we also save Precision and Recall. mAP may not be sufficient solely.

Reply:
Thx for the contribution, this can be helpful, even this is not in the scope of this task, I would say we can create a follow-up task for it.

Reply:
follow-up task created: https://wesleyclover.atlassian.net/browse/PRTD-5351?atlOrigin=eyJpIjoiODZiNzhjYWZiMDRkNGQ0MDg2NWI0MjM3NWIxZjE4ZjciLCJwIjoiaiJ9